### ISSUE TITLE
How to line break - Referring to CLOSED bug, "how to line break ? #273"

### ISSUE THREAD

**Issue by sawdit** [remarkjs/react-markdown#561]
I'm not sure how bug #273 got closed as nothing worked for me. However, reading between the lines I took the example from @JulesPatry and modified it until it worked for me.

Jules - `const source = value.replace(/\\n/gi, '\n &nbsp;');` - Didn't work

Mine - `const source = value.replace(/\\n/gi, '\n \n');` - Worked

Hitting Double Enter in the <textarea> didn't work out of the box. Must be something to do with the way I'm storing or retrieving it within MongoDB.

Like I said, this was the only thing that would work for me. Everything else (Double Space, Double Enter, Double Space Single Enter...) Failed

**Comment by wooorm** [comment]
Use backslash before a newline for CommonMark breaks, or use remark-breaks for any newline as a break

**Comment by sawdit** [comment]
I spent (more time than I should have) trying to get it to work. Then I applied my fix which work. On getting the code ready to try both of your suggestions, i.e feeding the data directly in, it started working of it's own accord???

I don't understand what has changed or is different from when I first implemented this code! Further more, another fix I had in place also isn't required an longer and now the expected markdown is displaying as expected.

I don't... All part of the learning curve I guess :-)

@woorm Thanks for the speedy reply.
